packages/web-components/src/components/cbp-toast/cbp-toast.stories.tsx

```diff
@@ -41,16 +42,18 @@ export default {
   export const Toast = Template.bind({});
   
   Toast.args = {
+    open: true,
     icon: 'user',
     content: 'Notification Description - A rule you are following just fired.',
```

Comment:
You should have a control for the title as well. This is hardcoded into your stories.

Reply:
added control for title